lab-tara/src/ciphers/CaesarShiftCipher.java

```diff
@@ -0,0 +1,38 @@
+package ciphers;
+
+public class CaesarShiftCipher extends Cipher {
+
+    public static String encrypt (String payload, int shift) {
+        char[] arr = payload.toCharArray();
+        int ascii;

```

Comment:
You don't need to worry about ascii for this problem and it isn't the best way to handle things.  You should be using your `ALPHABET ` that is created in `Cipher`, instead.